model/menu.js

```diff
@@ -1,13 +1,14 @@
-//Kyler Dotson
 'use strict';
 
 const Mongoose = require('mongoose');
+const Schema = mongoose.Schema;
```

Comment:
Alternatively, make the first mongoose lowercase (which is what is MUCH more common).